src/controllers/users.controller.ts

```diff
@@ -48,7 +49,7 @@ export class UserController {
     }
   }
 
-  @Authorized()
+  @Authorized(Roles.Admin)
```

Comment:
I do not have the trello so I may be mistaken, but should not the user be able to edit his own profile ?

Reply:
Yes, but this method allows to edit any user, which is an admin functionality. I need to add a method (or edit this one) to allow a user to edit his own profile, and it needs to be more limited

Reply:
You are right, did not see it was not limited for the logged in user.